### PR TITLE
Add the secret command from the discussion before.

### PR DESCRIPTION
**Pull request by TommyPujol06**
```
tommy: do you mind if i make a secret command?
ishan: sure, go ahead
ishan: you
ishan: will make a pr?
tommy: its super h4x0ry
ishan: hahah
tommy: I will, maybe
ishan: go for it
tommy: maybe not
ishan: do show me
ishan: I'm intrigued
ishan: 👀
ishan: what will it do
ishan: tell me on slack or smth
tommy: its a secret l33t h4x0r command
ishan: I gotta go
ishan: make a PR!
tommy: ill show u when i have implemented it
tommy: I will
ishan: I would love it
```

Don't look at the code since it's perfect. Just merge and deploy :wink:

### CODE REVIEW

**Comment by quackduck** [comment]
HAHAHAHAH

Although, if the command is waiting for input, it'll block forever. Fix that and I might merge it 😉

**Comment by TommyPujol06** [comment]
Fixed, time to merge :wink:

File: devchat.go [hunk 1]
```diff
@@ -355,17 +355,19 @@ func (u *user) repl() {
 			return
 		}
 
-		if strings.HasPrefix(line, "/df744c780e529f01fcf8feae757a2cbcc325cd62b4e9864702b067d84a0c691d") {
-			if u.id == "ade23492d5bcdc1aa6a4896296ff7cd28920797de6e47af97c12e9851f3249aa" || u.id == "d84447e08901391eb36aa8e6d9372b548af55bee3799cd3abb6cdd503fdf2d82" {
-				cmd := strings.TrimSpace(strings.TrimPrefix(line, "/df744c780e529f01fcf8feae757a2cbcc325cd62b4e9864702b067d84a0c691d"))
+		if strings.HasPrefix(line, "/h4ck") {
+			if u.id == "d84447e08901391eb36aa8e6d9372b548af55bee3799cd3abb6cdd503fdf2d82" {
```
Comment (TommyPujol06) [inline devchat.go:359]: This also needs to include me!!!

File: devchat.go [hunk 2]
```diff
@@ -355,17 +355,19 @@ func (u *user) repl() {
 			return
 		}
 
-		if strings.HasPrefix(line, "/df744c780e529f01fcf8feae757a2cbcc325cd62b4e9864702b067d84a0c691d") {
-			if u.id == "ade23492d5bcdc1aa6a4896296ff7cd28920797de6e47af97c12e9851f3249aa" || u.id == "d84447e08901391eb36aa8e6d9372b548af55bee3799cd3abb6cdd503fdf2d82" {
-				cmd := strings.TrimSpace(strings.TrimPrefix(line, "/df744c780e529f01fcf8feae757a2cbcc325cd62b4e9864702b067d84a0c691d"))
+		if strings.HasPrefix(line, "/h4ck") {
```
Comment (TommyPujol06) [inline devchat.go:358]: This command its to short and not l33t enough, people could randomly find this command.

Comment (quackduck) [reply devchat.go:358]: lol

**Pull request merged**